src/router.rs

```diff
@@ -11,13 +12,20 @@ use iron::Url;
 use recognizer::Router as Recognizer;
 use recognizer::{Match, Params};
 
+
+pub struct RouterInner {
```

Comment:
The reason this is `pub` is because otherwise I can't import it from `url_for.rs`